modin/test/test_docstring_urls.py

```diff
@@ -37,18 +38,18 @@ def doc_urls(get_generated_doc_urls):
 def test_all_urls_exist(doc_urls):
     broken = []
     # TODO: remove the hack after pandas fixes it
-    broken_urls = (
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.DataFrame.flags.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.Series.info.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.DataFrame.isetitem.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.Series.swapaxes.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.DataFrame.to_numpy.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.Series.axes.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.Series.divmod.html",
-        "https://pandas.pydata.org/pandas-docs/version/2.2.0/reference/api/pandas.Series.rdivmod.html",
+    methods_with_broken_urls = (
+        "pandas.DataFrame.flags",
+        "pandas.Series.info",
+        "pandas.DataFrame.isetitem",
+        "pandas.Series.swapaxes",
+        "pandas.DataFrame.to_numpy",
+        "pandas.Series.axes",
+        "pandas.Series.divmod",
+        "pandas.Series.rdivmod",
     )
-    for broken_url in broken_urls:
-        doc_urls.remove(broken_url)
+    for broken_method in methods_with_broken_urls:
+        doc_urls.remove(PANDAS_API_URL_TEMPLATE.format(broken_method))
```

Comment:
```suggestion
    for broken_url in methods_with_broken_urls:
        doc_urls.remove(PANDAS_API_URL_TEMPLATE.format(broken_url))
```

Reply:
Is it clearer?

Reply:
why? we're iterating over method names here, not urls